tests/manage/pv_services/pvc_clone/test_clone_when_pvc_full.py

```diff
@@ -91,6 +95,34 @@ def test_clone_when_full(self, pvc_clone_factory, pod_factory):
         cloned_pvcs = [pvc_clone_factory(pvc_obj) for pvc_obj in self.pvcs]
         log.info("Created clone of the PVCs. Cloned PVCs are Bound")
 
+        # Bug 2042318
+        for clone_pvc in cloned_pvcs:
+            if (
+                clone_pvc.get().get("spec").get("storageClassName")
+                == "ocs-storagecluster-cephfs"
+            ):
+                pv = clone_pvc.get().get("spec").get("volumeName")
+                func_calls = "clone"
+                error_msg = f"{pv} failed to create clone from subvolume"
```

Comment:
IIUC, the error "clone from snapshot is pending" is also relevant. This will ensure that the correct error is verified.